features/lorawan/LoRaWANInterface.cpp

```diff
@@ -0,0 +1,177 @@
+/**
```

Comment:
Is there any design docs available for this large change set. That can help with code reviewing?